BringgSDK.spec.js

```diff
@@ -434,4 +436,67 @@ describe('BringgSDK', function () {
       expect(BringgSDK.disconnect).not.toHaveBeenCalled();
     });
   });
+
+  describe('getDriverPhone', function(){
+    it('should fail if shared_uuid is not passed', function(){
+      window.$ = {get:function(){}};
+      spyOn(window.$, 'get');
```

Comment:
added jquery too in the karma.config. but this does not fix it. 
i moved the window.$ = {get:function(){}}; to the main before each